examples/analysis/ubet/compress_tdag.py

```diff
@@ -0,0 +1,85 @@
+from typing import List
```

Comment:
I know @ESultanik was planning to integrate this in a separate branch. We should probably synch with him as we had discussion about a potential bug that he probably fixed (`ceil` instead of `round`).

Reply:
The idea with including here was just to satisfy "if I were to dig into the polytracker codebase, is everything I need to reproduce the results in the ubet paper located there?" for a particular point-in-time! I will leave it and let @ESultanik delete this version when he has integrated it into the main polytracker python then.